### PR TITLE
[1.1] feat: add swapOnlyUsage in MemoryStats

### PR DESCRIPTION
**Pull request by harche**
This is a 1.1 backport of https://github.com/opencontainers/runc/pull/4010

### CODE REVIEW

**Review by kolyshkin — APPROVED** [review]
LGTM (but I hope we'll release 1.2.0 soon rather than make more 1.1.x releases)

**Review by mrunalp — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -46,6 +46,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
    This aligns cgroupv2 root usage more closely with cgroupv1 reporting.
    Additionally, report root swap usage as sum of swap and memory usage,
    aligned with v1 and existing non-root v2 reporting. (#3933)
+ * Add `swapOnlyUsage` in `MemoryStats`. This field reports swap-only usage.
```
Comment (lifubang) [inline CHANGELOG.md:49]: I think these change logs is wrong, it should be removed.

**Pull request merged**